### PR TITLE
[Synthetics] update README

### PR DESCRIPTION
**Pull request by dominiqueclarke**
Fixes elastic/kibana#99504

Update the README of Synthetics. We've opted not to document fields, but rather provide helpful text for familiarizing users with the product offering.

Screen Shot 2021-05-06 at 12 45 01 PM

### CODE REVIEW

File: packages/synthetics/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.0.5" # unreleased
+  changes:
+    - description: update README
+      type: enhancement # can be one of: enhancement, bugfix, breaking-change
+      link: https://github.com/elastic/integrations/pull/974
 - version: "0.0.4" # unreleased
```
Comment (mtojek) [inline packages/synthetics/changelog.yml:7]: Out of curiosity - is 0.0.4 released? Same question to other versions as comments are misleading.

Comment (dominiqueclarke) [reply packages/synthetics/changelog.yml:7]: @mtojek By released, do you mean in production. I assumed this meant released in production registry when I created these comments, but I should have asked for clarity. They are not release in the production registry.

File: packages/synthetics/docs/README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+React to availability issues across your apps and services before they affect users.
 
-Run lightweight synthetic checks on a schedule to ensure your HTTP, TCP, and ICMP services are up. Pairs with the Uptime Kibana UI.
+While a service being "up" or "down" might seem binary, the data surrounding each data point isn't. Give your team the context they need to go well beyond status codes. With Elastic Uptime, powered by open source [Heartbeat](https://www.elastic.co/beats/heartbeat), your availability data works in concert with rich context provided by [logs](https://www.elastic.co/log-monitoring), [metrics](https://www.elastic.co/infrastructure-monitoring), and [APM](https://www.elastic.co/apm) — making it simpler to connect the dots, correlate activity, and solve problems quickly.
 
+Organizations need to track the performance and availability of a rapidly expanding list of components and services. Investigating the inevitable question, "Is X down?" is streamlined with Elastic Uptime. Active monitoring with [custom alerting options](https://www.elastic.co/what-is/kibana-alerting) keeps you ahead of your users — ensuring they have a positive, issue-free experience.
```
Comment (justinkambic) [inline packages/synthetics/docs/README.md:5]:
```suggestion
Organizations need to track the performance and availability of a rapidly expanding list of components and services. Investigating the inevitable question, "Is it down?" is streamlined with Elastic Uptime. Active monitoring with [custom alerting options](https://www.elastic.co/what-is/kibana-alerting) keeps you ahead of your users — ensuring they have a positive, issue-free experience.
```

I think this copy sounds a little nicer, but I'm guessing this already came from product and they picked these words carefully, so it's not something worth worrying about.

**Review by justinkambic — APPROVED** [review]
LGTM

**Pull request merged**